### ISSUE TITLE
Fails node-gyp "rebuild" dependency co-pg

### ISSUE THREAD

**Issue by c11z** [chilts/koa-pg#4]
I am trying to install the koa-pg library, but I can't get past the node-gyp "rebuild" of the pg dependency.

This is on a vagrant ubuntu 14.04 box

In preparation I installed nvm and node 0.11.14 then ran:

```
sudo apt-get -y install libpq-dev
npm install -g node-gyp
```

Trying to install koa-pg locally I get this error:

```
vagrant@vagrant-ubuntu-trusty-64:/vagrant$ npm install koa-pg
\
> pg@2.8.5 install /vagrant/node_modules/koa-pg/node_modules/co-pg/node_modules/pg
> node-gyp rebuild || (exit 0)

child_process: customFds option is deprecated, use stdio instead.
make: Entering directory `/vagrant/node_modules/koa-pg/node_modules/co-pg/node_modules/pg/build'
  CXX(target) Release/obj.target/binding/src/binding.o
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h: In function ‘v8::Handle<v8::Value> NanError(const char*)’:
../node_modules/nan/nan.h:298:38: error: ‘New’ is not a member of ‘v8::String’
 # define _NAN_ERROR(fun, errmsg) fun(v8::String::New(errmsg))
                                      ^
../node_modules/nan/nan.h:318:13: note: in expansion of macro ‘_NAN_ERROR’
     return  _NAN_ERROR(v8::Exception::Error, errmsg);
             ^
../node_modules/nan/nan.h: In function ‘void NanThrowError(const char*)’:
../node_modules/nan/nan.h:303:7: error: ‘ThrowException’ is not a member of ‘v8’
       v8::ThrowException(_NAN_ERROR(fun, errmsg));                             \
       ^
../node_modules/nan/nan.h:322:5: note: in expansion of macro ‘_NAN_THROW_ERROR’
     _NAN_THROW_ERROR(v8::Exception::Error, errmsg);
     ^
../node_modules/nan/nan.h:298:38: error: ‘New’ is not a member of ‘v8::String’
 # define _NAN_ERROR(fun, errmsg) fun(v8::String::New(errmsg))
                                      ^
../node_modules/nan/nan.h:303:26: note: in expansion of macro ‘_NAN_ERROR’
       v8::ThrowException(_NAN_ERROR(fun, errmsg));                             \
                          ^
../node_modules/nan/nan.h:322:5: note: in expansion of macro ‘_NAN_THROW_ERROR’
     _NAN_THROW_ERROR(v8::Exception::Error, errmsg);
     ^
../node_modules/nan/nan.h: In function ‘void NanThrowError(v8::Handle<v8::Value>)’:
../node_modules/nan/nan.h:327:5: error: ‘ThrowException’ is not a member of ‘v8’
     v8::ThrowException(error);
     ^
../node_modules/nan/nan.h: In function ‘v8::Handle<v8::Value> NanError(const char*, int)’:
../node_modules/nan/nan.h:334:53: error: ‘New’ is not a member of ‘v8::String’
     v8::Local<v8::Value> err = v8::Exception::Error(v8::String::New(msg));
                                                     ^
../node_modules/nan/nan.h:336:14: error: ‘New’ is not a member of ‘v8::String’
     obj->Set(v8::String::New("code"), v8::Int32::New(errorNumber));
              ^
../node_modules/nan/nan.h:336:65: error: no matching function for call to ‘v8::Int32::New(const int&)’
     obj->Set(v8::String::New("code"), v8::Int32::New(errorNumber));
                                                                 ^
../node_modules/nan/nan.h:336:65: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note: static v8::Local<v8::Integer> v8::Integer::New(v8::Isolate*, int32_t)
   static Local<Integer> New(Isolate* isolate, int32_t value);
                         ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note:   candidate expects 2 arguments, 1 provided
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h: In function ‘v8::Handle<v8::Value> NanTypeError(const char*)’:
../node_modules/nan/nan.h:298:38: error: ‘New’ is not a member of ‘v8::String’
 # define _NAN_ERROR(fun, errmsg) fun(v8::String::New(errmsg))
                                      ^
../node_modules/nan/nan.h:348:12: note: in expansion of macro ‘_NAN_ERROR’
     return _NAN_ERROR(v8::Exception::TypeError, errmsg);
            ^
../node_modules/nan/nan.h: In function ‘void NanThrowTypeError(const char*)’:
../node_modules/nan/nan.h:303:7: error: ‘ThrowException’ is not a member of ‘v8’
       v8::ThrowException(_NAN_ERROR(fun, errmsg));                             \
       ^
../node_modules/nan/nan.h:352:5: note: in expansion of macro ‘_NAN_THROW_ERROR’
     _NAN_THROW_ERROR(v8::Exception::TypeError, errmsg);
     ^
../node_modules/nan/nan.h:298:38: error: ‘New’ is not a member of ‘v8::String’
 # define _NAN_ERROR(fun, errmsg) fun(v8::String::New(errmsg))
                                      ^
../node_modules/nan/nan.h:303:26: note: in expansion of macro ‘_NAN_ERROR’
       v8::ThrowException(_NAN_ERROR(fun, errmsg));                             \
                          ^
../node_modules/nan/nan.h:352:5: note: in expansion of macro ‘_NAN_THROW_ERROR’
     _NAN_THROW_ERROR(v8::Exception::TypeError, errmsg);
     ^
../node_modules/nan/nan.h: In function ‘v8::Handle<v8::Value> NanRangeError(const char*)’:
../node_modules/nan/nan.h:298:38: error: ‘New’ is not a member of ‘v8::String’
 # define _NAN_ERROR(fun, errmsg) fun(v8::String::New(errmsg))
                                      ^
../node_modules/nan/nan.h:356:12: note: in expansion of macro ‘_NAN_ERROR’
     return _NAN_ERROR(v8::Exception::RangeError, errmsg);
            ^
../node_modules/nan/nan.h: In function ‘void NanThrowRangeError(const char*)’:
../node_modules/nan/nan.h:303:7: error: ‘ThrowException’ is not a member of ‘v8’
       v8::ThrowException(_NAN_ERROR(fun, errmsg));                             \
       ^
../node_modules/nan/nan.h:360:5: note: in expansion of macro ‘_NAN_THROW_ERROR’
     _NAN_THROW_ERROR(v8::Exception::RangeError, errmsg);
     ^
../node_modules/nan/nan.h:298:38: error: ‘New’ is not a member of ‘v8::String’
 # define _NAN_ERROR(fun, errmsg) fun(v8::String::New(errmsg))
                                      ^
../node_modules/nan/nan.h:303:26: note: in expansion of macro ‘_NAN_ERROR’
       v8::ThrowException(_NAN_ERROR(fun, errmsg));                             \
                          ^
../node_modules/nan/nan.h:360:5: note: in expansion of macro ‘_NAN_THROW_ERROR’
     _NAN_THROW_ERROR(v8::Exception::RangeError, errmsg);
     ^
../node_modules/nan/nan.h: In constructor ‘NanCallback::NanCallback(const v8::Local<v8::Function>&)’:
../node_modules/nan/nan.h:668:49: error: no matching function for call to ‘v8::Object::New()’
     v8::Local<v8::Object> obj = v8::Object::New();
                                                 ^
../node_modules/nan/nan.h:668:49: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2372:24: note: static v8::Local<v8::Object> v8::Object::New(v8::Isolate*)
   static Local<Object> New(Isolate* isolate);
                        ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2372:24: note:   candidate expects 1 argument, 0 provided
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../node_modules/nan/nan.h:669:14: note: in expansion of macro ‘NanSymbol’
     obj->Set(NanSymbol("callback"), fn);
              ^
../node_modules/nan/nan.h: In destructor ‘NanCallback::~NanCallback()’:
../node_modules/nan/nan.h:675:12: error: ‘class v8::Persistent<v8::Object>’ has no member named ‘Dispose’
     handle.Dispose();
            ^
../node_modules/nan/nan.h:676:12: error: ‘class v8::Persistent<v8::Object>’ has no member named ‘Clear’
     handle.Clear();
            ^
../node_modules/nan/nan.h: In member function ‘v8::Local<v8::Function> NanCallback::GetFunction()’:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../node_modules/nan/nan.h:680:46: note: in expansion of macro ‘NanSymbol’
     return NanPersistentToLocal(handle)->Get(NanSymbol("callback"))
                                              ^
../node_modules/nan/nan.h:681:25: error: expected primary-expression before ‘>’ token
         .As<v8::Function>();
                         ^
../node_modules/nan/nan.h:681:27: error: expected primary-expression before ‘)’ token
         .As<v8::Function>();
                           ^
../node_modules/nan/nan.h: In member function ‘void NanCallback::Call(int, v8::Local<v8::Value>*)’:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../node_modules/nan/nan.h:693:13: note: in expansion of macro ‘NanSymbol’
         Get(NanSymbol("callback")).As<v8::Function>();
             ^
../node_modules/nan/nan.h:693:51: error: expected primary-expression before ‘>’ token
         Get(NanSymbol("callback")).As<v8::Function>();
                                                   ^
../node_modules/nan/nan.h:693:53: error: expected primary-expression before ‘)’ token
         Get(NanSymbol("callback")).As<v8::Function>();
                                                     ^
../node_modules/nan/nan.h:695:9: error: ‘GetCurrent’ is not a member of ‘v8::Context’
         v8::Context::GetCurrent()->Global()
         ^
../node_modules/nan/nan.h: In member function ‘void NanAsyncWorker::SavePersistent(const char*, v8::Local<v8::Object>&)’:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../node_modules/nan/nan.h:739:17: note: in expansion of macro ‘NanSymbol’
     handle->Set(NanSymbol(key), obj);
                 ^
../node_modules/nan/nan.h: In member function ‘v8::Local<v8::Object> NanAsyncWorker::GetFromPersistent(const char*)’:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../node_modules/nan/nan.h:746:24: note: in expansion of macro ‘NanSymbol’
     return handle->Get(NanSymbol(key)).As<v8::Object>();
                        ^
../node_modules/nan/nan.h:746:53: error: expected primary-expression before ‘>’ token
     return handle->Get(NanSymbol(key)).As<v8::Object>();
                                                     ^
../node_modules/nan/nan.h:746:55: error: expected primary-expression before ‘)’ token
     return handle->Get(NanSymbol(key)).As<v8::Object>();
                                                       ^
../node_modules/nan/nan.h: In member function ‘virtual void NanAsyncWorker::HandleErrorCallback()’:
../node_modules/nan/nan.h:768:30: error: ‘New’ is not a member of ‘v8::String’
         v8::Exception::Error(v8::String::New(errmsg))
                              ^
../node_modules/nan/nan.h: In function ‘char* NanFromV8String(v8::Handle<v8::Value>, Nan::Encoding, size_t*, char*, size_t, int)’:
../node_modules/nan/nan.h:987:3: error: ‘AsciiValue’ is not a member of ‘v8::String’
   v8::String::AsciiValue value(toStr);
   ^
../node_modules/nan/nan.h:987:26: error: expected ‘;’ before ‘value’
   v8::String::AsciiValue value(toStr);
                          ^
../node_modules/nan/nan.h:1050:39: error: ‘value’ was not declared in this scope
       sz_ = _nan_base64_decoded_size(*value, toStr->Length());
                                       ^
../src/binding.cc: In static member function ‘static void Connection::Init(v8::Handle<v8::Object>)’:
../src/binding.cc:33:58: error: no matching function for call to ‘v8::FunctionTemplate::New(void (&)(const v8::FunctionCallbackInfo<v8::Value>&))’
     Local<FunctionTemplate> t = FunctionTemplate::New(New);
                                                          ^
../src/binding.cc:33:58: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:3434:34: note: static v8::Local<v8::FunctionTemplate> v8::FunctionTemplate::New(v8::Isolate*, v8::FunctionCallback, v8::Handle<v8::Value>, v8::Handle<v8::Signature>, int)
   static Local<FunctionTemplate> New(
                                  ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:3434:34: note:   no known conversion for argument 1 from ‘void(const v8::FunctionCallbackInfo<v8::Value>&)’ to ‘v8::Isolate*’
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:36:21: note: in expansion of macro ‘NanSymbol’
     t->SetClassName(NanSymbol("Connection"));
                     ^
../src/binding.cc:51:17: error: ‘NewSymbol’ is not a member of ‘v8::String’
     target->Set(String::NewSymbol("Connection"), t->GetFunction());
                 ^
../src/binding.cc: In static member function ‘static void Connection::EscapeIdentifier(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/binding.cc:124:26: error: ‘New’ is not a member of ‘v8::String’
     Local<Value> jsStr = String::New(escapedStr, strlen(escapedStr));
                          ^
../src/binding.cc: In static member function ‘static void Connection::EscapeLiteral(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/binding.cc:149:26: error: ‘New’ is not a member of ‘v8::String’
     Local<Value> jsStr = String::New(escapedStr, strlen(escapedStr));
                          ^
../src/binding.cc: In member function ‘void Connection::HandleNotice(const char*)’:
../src/binding.cc:480:28: error: ‘New’ is not a member of ‘v8::String’
     Handle<Value> notice = String::New(message);
                            ^
../src/binding.cc: In member function ‘void Connection::HandleIOEvent(int)’:
../src/binding.cc:539:44: error: no matching function for call to ‘v8::Object::New()’
         Local<Object> result = Object::New();
                                            ^
../src/binding.cc:539:44: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2372:24: note: static v8::Local<v8::Object> v8::Object::New(v8::Isolate*)
   static Local<Object> New(Isolate* isolate);
                        ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2372:24: note:   candidate expects 1 argument, 0 provided
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:540:21: note: in expansion of macro ‘NanSymbol’
         result->Set(NanSymbol("channel"), String::New(notify->relname));
                     ^
../src/binding.cc:540:43: error: ‘New’ is not a member of ‘v8::String’
         result->Set(NanSymbol("channel"), String::New(notify->relname));
                                           ^
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:541:21: note: in expansion of macro ‘NanSymbol’
         result->Set(NanSymbol("payload"), String::New(notify->extra));
                     ^
../src/binding.cc:541:43: error: ‘New’ is not a member of ‘v8::String’
         result->Set(NanSymbol("payload"), String::New(notify->extra));
                                           ^
../src/binding.cc: In member function ‘void Connection::EmitRowDescription(const PGresult*)’:
../src/binding.cc:588:35: error: no matching function for call to ‘v8::Array::New()’
     Local<Array> row = Array::New();
                                   ^
../src/binding.cc:588:35: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2401:23: note: static v8::Local<v8::Array> v8::Array::New(v8::Isolate*, int)
   static Local<Array> New(Isolate* isolate, int length = 0);
                       ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2401:23: note:   candidate expects 2 arguments, 0 provided
../src/binding.cc:591:41: error: no matching function for call to ‘v8::Object::New()’
       Local<Object> field = Object::New();
                                         ^
../src/binding.cc:591:41: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2372:24: note: static v8::Local<v8::Object> v8::Object::New(v8::Isolate*)
   static Local<Object> New(Isolate* isolate);
                        ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2372:24: note:   candidate expects 1 argument, 0 provided
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:594:18: note: in expansion of macro ‘NanSymbol’
       field->Set(NanSymbol("name"), String::New(fieldName));
                  ^
../src/binding.cc:594:37: error: ‘New’ is not a member of ‘v8::String’
       field->Set(NanSymbol("name"), String::New(fieldName));
                                     ^
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:598:18: note: in expansion of macro ‘NanSymbol’
       field->Set(NanSymbol("dataTypeID"), Integer::New(fieldType));
                  ^
../src/binding.cc:598:65: error: no matching function for call to ‘v8::Integer::New(int&)’
       field->Set(NanSymbol("dataTypeID"), Integer::New(fieldType));
                                                                 ^
../src/binding.cc:598:65: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note: static v8::Local<v8::Integer> v8::Integer::New(v8::Isolate*, int32_t)
   static Local<Integer> New(Isolate* isolate, int32_t value);
                         ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note:   candidate expects 2 arguments, 1 provided
../src/binding.cc:600:40: error: no matching function for call to ‘v8::Integer::New(int&)’
       row->Set(Integer::New(fieldNumber), field);
                                        ^
../src/binding.cc:600:40: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note: static v8::Local<v8::Integer> v8::Integer::New(v8::Isolate*, int32_t)
   static Local<Integer> New(Isolate* isolate, int32_t value);
                         ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note:   candidate expects 2 arguments, 1 provided
../src/binding.cc: In member function ‘void Connection::EmitCommandMetaData(PGresult*)’:
../src/binding.cc:661:38: error: no matching function for call to ‘v8::Object::New()’
     Local<Object> info = Object::New();
                                      ^
../src/binding.cc:661:38: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2372:24: note: static v8::Local<v8::Object> v8::Object::New(v8::Isolate*)
   static Local<Object> New(Isolate* isolate);
                        ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2372:24: note:   candidate expects 1 argument, 0 provided
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:662:15: note: in expansion of macro ‘NanSymbol’
     info->Set(NanSymbol("command"), String::New(PQcmdStatus(result)));
               ^
../src/binding.cc:662:37: error: ‘New’ is not a member of ‘v8::String’
     info->Set(NanSymbol("command"), String::New(PQcmdStatus(result)));
                                     ^
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:663:15: note: in expansion of macro ‘NanSymbol’
     info->Set(NanSymbol("value"), String::New(PQcmdTuples(result)));
               ^
../src/binding.cc:663:35: error: ‘New’ is not a member of ‘v8::String’
     info->Set(NanSymbol("value"), String::New(PQcmdTuples(result)));
                                   ^
../src/binding.cc: In member function ‘void Connection::HandleTuplesResult(const PGresult*)’:
../src/binding.cc:678:37: error: no matching function for call to ‘v8::Array::New()’
       Local<Array> row = Array::New();
                                     ^
../src/binding.cc:678:37: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2401:23: note: static v8::Local<v8::Array> v8::Array::New(v8::Isolate*, int)
   static Local<Array> New(Isolate* isolate, int length = 0);
                       ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:2401:23: note:   candidate expects 2 arguments, 0 provided
../src/binding.cc:684:44: error: no matching function for call to ‘v8::Integer::New(int&)’
           row->Set(Integer::New(fieldNumber), Null());
                                            ^
../src/binding.cc:684:44: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note: static v8::Local<v8::Integer> v8::Integer::New(v8::Isolate*, int32_t)
   static Local<Integer> New(Isolate* isolate, int32_t value);
                         ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note:   candidate expects 2 arguments, 1 provided
../src/binding.cc:684:52: error: too few arguments to function ‘v8::Handle<v8::Primitive> v8::Null(v8::Isolate*)’
           row->Set(Integer::New(fieldNumber), Null());
                                                    ^
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:306:28: note: declared here
   friend Handle<Primitive> Null(Isolate* isolate);
                            ^
../src/binding.cc:687:44: error: no matching function for call to ‘v8::Integer::New(int&)’
           row->Set(Integer::New(fieldNumber), String::New(fieldValue));
                                            ^
../src/binding.cc:687:44: note: candidate is:
In file included from /home/vagrant/.node-gyp/0.11.14/src/node.h:61:0,
                 from ../node_modules/nan/nan.h:112,
                 from ../src/binding.cc:3:
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note: static v8::Local<v8::Integer> v8::Integer::New(v8::Isolate*, int32_t)
   static Local<Integer> New(Isolate* isolate, int32_t value);
                         ^
/home/vagrant/.node-gyp/0.11.14/deps/v8/include/v8.h:1989:25: note:   candidate expects 2 arguments, 1 provided
../src/binding.cc:687:47: error: ‘New’ is not a member of ‘v8::String’
           row->Set(Integer::New(fieldNumber), String::New(fieldValue));
                                               ^
../src/binding.cc: In member function ‘void Connection::HandleErrorResult(const PGresult*)’:
../src/binding.cc:707:62: error: ‘New’ is not a member of ‘v8::String’
     Local<Object> msg = Local<Object>::Cast(Exception::Error(String::New(errorMessage)));
                                                              ^
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:710:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("severity"), PG_DIAG_SEVERITY);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:711:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("code"), PG_DIAG_SQLSTATE);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:712:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("detail"), PG_DIAG_MESSAGE_DETAIL);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:713:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("hint"), PG_DIAG_MESSAGE_HINT);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:714:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("position"), PG_DIAG_STATEMENT_POSITION);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:715:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("internalPosition"), PG_DIAG_INTERNAL_POSITION);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:716:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("internalQuery"), PG_DIAG_INTERNAL_QUERY);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:717:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("where"), PG_DIAG_CONTEXT);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:718:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("file"), PG_DIAG_SOURCE_FILE);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:719:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("line"), PG_DIAG_SOURCE_LINE);
                                   ^
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:720:35: note: in expansion of macro ‘NanSymbol’
     AttachErrorField(result, msg, NanSymbol("routine"), PG_DIAG_SOURCE_FUNCTION);
                                   ^
../src/binding.cc: In member function ‘void Connection::AttachErrorField(const PGresult*, v8::Local<v8::Object>, v8::Local<v8::String>, int)’:
../src/binding.cc:731:24: error: ‘New’ is not a member of ‘v8::String’
       msg->Set(symbol, String::New(val));
                        ^
../src/binding.cc: In member function ‘void Connection::Emit(const char*)’:
../src/binding.cc:749:31: error: ‘New’ is not a member of ‘v8::String’
     Handle<Value> args[1] = { String::New(message) };
                               ^
../src/binding.cc: In member function ‘void Connection::Emit(const char*, v8::Handle<v8::Value>*)’:
../src/binding.cc:755:31: error: ‘New’ is not a member of ‘v8::String’
     Handle<Value> args[2] = { String::New(message), *arg };
                               ^
In file included from ../src/binding.cc:3:0:
../src/binding.cc: In member function ‘void Connection::Emit(int, v8::Handle<v8::Value>*)’:
../node_modules/nan/nan.h:159:26: error: ‘NewSymbol’ is not a member of ‘v8::String’
 #define NanSymbol(value) v8::String::NewSymbol(value)
                          ^
../src/binding.cc:762:58: note: in expansion of macro ‘NanSymbol’
     Local<Value> emit_v = NanObjectWrapHandle(this)->Get(NanSymbol("emit"));
                                                          ^
../src/binding.cc: In member function ‘void Connection::EmitError(const char*)’:
../src/binding.cc:805:47: error: ‘New’ is not a member of ‘v8::String’
     Local<Value> exception = Exception::Error(String::New(message));
                                               ^
In file included from ../src/binding.cc:3:0:
../node_modules/nan/nan.h: In instantiation of ‘void NanDispose(v8::Persistent<T, v8::NonCopyablePersistentTraits<T> >&) [with T = v8::Object]’:
../node_modules/nan/nan.h:717:34:   required from here
../node_modules/nan/nan.h:373:5: error: ‘class v8::Persistent<v8::Object>’ has no member named ‘Clear’
     handle.Clear();
     ^
make: *** [Release/obj.target/binding/src/binding.o] Error 1
make: Leaving directory `/vagrant/node_modules/koa-pg/node_modules/co-pg/node_modules/pg/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/vagrant/.nvm/v0.11.14/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1059:12)
gyp ERR! System Linux 3.13.0-39-generic
gyp ERR! command "node" "/home/vagrant/.nvm/v0.11.14/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /vagrant/node_modules/koa-pg/node_modules/co-pg/node_modules/pg
gyp ERR! node -v v0.11.14
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok
koa-pg@0.23.0 node_modules/koa-pg
└── co-pg@0.1.1 (thunkify@0.0.1, pg@2.8.5)
```

I can install pg and co-pg separately with no issues. I'm pretty new to node, am I doing something dumb?

**Comment by chilts** [comment]
Does this still happen? How about on the recently released v0.24? Thanks.

**Comment by c11z** [comment]
Cleaning up
